R/vignette.R

```diff
@@ -56,10 +57,24 @@ use_vignette_template <- function(template, name, title) {
   data$braced_vignette_title <- glue::glue("{{{title}}}")
 
   use_template(template,
-    save_as = path,
-    data = data,
-    open = TRUE
+               save_as = path,
+               data = data,
+               open = TRUE
```

Comment:
This whitespace looks odd. styler is a good package to use re: applying tidyverse "house style".

Reply:
Ah! Sorry about that. Used to just cleaning spacing w/ cmd + i , didn't realize that would take out of tidyverse style. Fixed with styler.